n2kMapper.js

```diff
@@ -16,8 +16,26 @@ Object.assign(n2kMappings, require('./actisense'))
 Object.assign(n2kMappings, require('./digitalyacht'))
 
 function N2kMapper (options) {
-  this.state = {}
+  this.state = {  }
   this.unknownPGNs = {}
+  this.customPgns = {}
+  this.options = options || {}
+  
+  if ( this.options.onPropertyValues ) {
+    this.options.onPropertyValues('pgn-to-signalk', values => {
+      values.filter(v => v != null).forEach(pv => {
```

Comment:
This is to discard the first value `undefined`, right? I think the test should be `!== undefined`, as this is not about null and null comparisons are needlessly tricky:

```
undefined != null
false
undefined !== null
true
```

Reply:
fixed